package.json

```diff
@@ -36,7 +36,7 @@
     "compile": "lingui compile"
   },
   "dependencies": {
-    "@funboxteam/crafter": "5.0.1",
+    "@funboxteam/crafter": "6.0.0",
```

Comment:
I had almost pressed the "Merge" button, but then I thought that we could update Crafter to the fresh-baked major version since we have a major release in Blueprinter too.